inference-engine/src/vpu/common/include/vpu/ngraph/operations/static_shape_loop.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/opsets/opset5.hpp>
+
+namespace ngraph { namespace vpu { namespace op {
+
+class StaticShapeLoop : public ngraph::opset5::Loop {
```

Comment:
I guess we should use opset6 for 2021.3 Release

Reply:
Nice catch! Thanks!

Reply:
Fixed